packages/interfaces/src/relay.ts

```diff
@@ -16,6 +16,7 @@ export interface IRelayAPI {
   readonly pubsubTopics: Set<PubsubTopic>;
   readonly gossipSub: GossipSub;
   start: () => Promise<void>;
+  waitForPeer: () => Promise<void>;
```

Comment:
```suggestion
  waitForPeers: () => Promise<void>;
```

Reply:
indeed it is a good option, using it
cc @adklempner